### DISCUSSION TITLE
Previously placed wire drops to ground when equipping a new wire

### DISCUSSION THREAD

**Discussion by Scaperjet** [FakeFishGames/Barotrauma#14076] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

My friend and I in a private server started a campaign, we had no issues until after passing the first locked checkpoint of the game, and purchasing a new tier 2 submarine, the Big Rig. I was attempting to wire up our new reactor, when I encountered the bug. It happens on any object we try to wire, and it happens to both of us. We've tried verifying files on steam, restarting game, switching back to our first sub, and ensuring we had wiring enabled in server settings. We should have been able to wire without any issues.

### Reproduction steps

1. Load into our multiplayer campaign.
2. Wire any object to another object
3. reequip any wire from hotbar
4. Previously placed wire drops to ground, removing itself from objects. 

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by James-Thegiant** [comment]
Had a very similar issue, additionally noticed that the wire appears to remain "in my hand" in the inventory, and the number of wires in my hand doesn't go down after placement, however all the signals carried by that wire appear to work until it is disconnected by either trying to place a new wire or switch out what you're holding in your hand. Sometimes it seemed to just stay in your hand after disconnecting, but sometimes it would drop to the ground.

Might be the same issue as reported in - [#15941](https://github.com/FakeFishGames/Barotrauma/discussions/15941): "Wires from non-player submarines are potentially bugged and cause normal wires to bug out as well"

Ours was a sub that we had just recently transferred our items to from a previous sub, so it's possible that is related, but I also have definitely taken wires from NPC subs previously so perhaps the bugged wires were already in our storage. No mods, multiplayer server hosted from in-game menu and all on Windows.

**Comment by BryceKKelly** [comment]
I have the exact same issue. Running a dedicated server. I also think there's some credibility to what @James-Thegiant said, because after reading that comment, I just switched from blue wires to green wires and did not get this issue, after getting it 5 times in a row from blue wires.

One caveat - I cannot confidently say that this issue was 100% consistent for me. I was not able to wire doors yesterday, but today I could. Might be because I'm using different wires and don't realise, might be because it's a desync issue that isn't always going to happen the same every time.
